bin/deploy

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
+set -o pipefail
+
 environment=$1
 revision=$2
 tag=$3
 cluster=$4
 
 sops --decrypt $(pwd)/config/deploy/${environment}/secrets.yaml > $(pwd)/config/deploy/${environment}/secrets.dec.yaml
-REVISION=${revision} kubernetes-deploy \
+krane render \
+    --filenames=$(pwd)/config/deploy/templates \
+    --bindings "@$(pwd)/config/deploy/values.yaml" "@$(pwd)/config/deploy/${environment}/values.yaml" "@$(pwd)/config/deploy/${environment}/secrets.dec.yaml" "image_tag=${tag}" "environment=${environment}" \
```

Comment:
```suggestion
    --bindings="@$(pwd)/config/deploy/values.yaml" "@$(pwd)/config/deploy/${environment}/values.yaml" "@$(pwd)/config/deploy/${environment}/secrets.dec.yaml" "image_tag=${tag}" "environment=${environment}" \
```

Reply:
not entirely sure you need the quotes. Any reason to keep them?

Reply:
I just tried to keep the change small @viniciusgama